src/Middleware/Diagnostics/src/DeveloperExceptionPage/DeveloperExceptionPageMiddlewareImpl.cs

```diff
@@ -101,6 +101,18 @@ public async Task Invoke(HttpContext context)
         }
         catch (Exception ex)
         {
+            if (ex is OperationCanceledException && context.RequestAborted.IsCancellationRequested)
```

Comment:
@halter pointed out that we should also handle IOException's in a similar way when RequestAborted.IsCancellationRequested.

https://github.com/dotnet/aspnetcore/issues/44697#issuecomment-1421520530

```suggestion
            if ((ex is OperationCanceledException || ex is IOException)
                && context.RequestAborted.IsCancellationRequested)
```